app/src/main/java/com/dessalines/thumbkey/utils/Types.kt

```diff
@@ -101,6 +101,7 @@ enum class KeyboardLayout(val title: String, val index: Int) {
     ThumbKeyESv1("Thumb-Key español v1", 4),
     ThumbKeyEUv1("Thumb-Key euskara v1", 5),
     ThumbKeyFAv1("Thumb-Key فارسیv1", 6),
+    ThumbKeyHEv1("Thumb-Key עברית v1", 49),
```

Comment:
Add this at the end of the list. The app sorts them by name anyway.

Reply:
Done :)